stdlib/public/core/StringNormalization.swift

```diff
@@ -318,10 +317,9 @@ internal func _allocateBuffers(
   icuOutputBuffer: inout UnsafeMutableBufferPointer<UInt16>
 ) {
   let output = count * _Normalization._maxNFCExpansionFactor * _Normalization._maxUTF16toUTF8ExpansionFactor
-  let icuInput = count
```

Comment:
Same comment here: this binding makes explicit the relationship between `icuInput` and `icuOutput` that cannot be deduced when using the name `count`.